nodelet/src/nodelet.cpp

```diff
@@ -341,7 +356,11 @@ int
         ROS_INFO("Bond broken, exiting");
         goto shutdown;
       }
+#ifndef _WIN32
       usleep(100000);
+#else
+      Sleep(100);
```

Comment:
Is there a specific header to include to get this function ?

Reply:
ros/ros.h include ros/platform.h and platform.h include windows.h that has the declare Sleep()

Reply:
great, thanks for confirming